### PR TITLE
Connect to api

### PR DESCRIPTION
**Pull request by MAleemH**
**I have done the following:**

- Connected my app to Bookstore API.
- Used fetch with createAsyncThunk to make API calls.
- Fetched books from Bookstore API on when the component has rendered with useEffect.
- Modified reducer logic to include HTTP requests to the Bookstore API.

### CODE REVIEW

File: src/redux/books/booksSlice.js [hunk 1]
```diff
@@ -1,42 +1,70 @@
...
-      item_id: 'item2',
-      title: 'Anna Karenina',
-      author: 'Leo Tolstoy',
-      category: 'Fiction',
+// unique api url
+const api = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/eHu0AJvEHeHUitjoncwV/books';
+
+// fetching data from api
+export const fetchBooksData = createAsyncThunk('fetchBooksData', async () => {
+  const response = await fetch(api);
```
Comment (topeogunleye) [inline src/redux/books/booksSlice.js:9]: - [ ] As required by project requirements, I kindly suggest that all HTTP requests are mad with `axios`.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @MAleemH, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/booksSlice.js (hunk 1 above)
Comment (MAleemH) [reply src/redux/books/booksSlice.js:9]: Hello @topeogunleye 
I hope you are doing well. As mentioned in the requirements, we have a choice to use in fetch and axios.

Comment (topeogunleye) [reply src/redux/books/booksSlice.js:9]: Sorry, my bad @MAleemH, please kindly submit again and you'll be approved

Note To Next Reviewer: Please Approve the project.

Comment (MAleemH) [reply src/redux/books/booksSlice.js:9]: Thank you so much @topeogunleye

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @MAleemH, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented requirements.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/redux/books/booksSlice.js (hunk 1 above)
Comment (topeogunleye) [reply src/redux/books/booksSlice.js:9]: @MAleemHm, you're welcome

**Pull request merged**